### ISSUE TITLE
[Question] Handle admob node when switching Scene

### ISSUE THREAD

**Issue by LicoriceSoft** [cengiz-pz/godot-android-admob-plugin#34]
Hi. I'm using your great plugin and I'm impressed and eternally grateful, but I have a small difficulty and have a question.
  
I'm a developer working in a non-English speaking country. Please excuse my lack of English. 

The project I'm working on is Godot Engine 4.2.2. So I've been testing with the 2.1 admob plugin.

I found that the admob node must be included in the Main Scene for the node to function. 
In fact, when I tested it in android environment, it worked very well.

However, here is the specific issue:

In my project, the Main Scene is not the scene where the gameplay exists, it's just the intro screen of the game. So I added the admob node to the Main Scene, but I didn't do any initialization and banner loading here because it's not the scene that needs ads.

My gameplay takes place in the Game Scene, which is the next scene after the Main Scene, so I added the admob node here, initialized it, loaded the banner, and performed the Show command. 

After some time, no banner was displayed. 

I checked to see if I had missed or entered the ad ID incorrectly, but it was entered correctly. I also initialized the admob node in the Main Scene and loaded the banner(not doing show function in Main Scene), but the banner was not displayed in the Game Scene.

I was wondering if I need to do something special process to get the admob functionality to work even if the scenes are switched.

(I ask for your understanding that I used a translator for smooth communication.)

**Comment by LicoriceSoft** [comment]
I have solved the problem. so I closed the issue

Thankfully, the cause of this issue is my lack of sleep.

I was in the middle of working during the dawn,  wasn't in a good condition.

The problem solving method was to initialize the admob node in the main scene and then create the admob node in the game scene too. Then, I initialized the node, loaded ad in game scene. finally ad was displayed only in game scene.
